src/synapse/azext_synapse/_help.py

```diff
@@ -154,122 +163,165 @@
 type: command
 short-summary: Pause a SQL pool.
-long-summary: Pause a SQL pool with SQL pool name.
 examples:
   - name: Pause SQL pool.
     text: |-
-        az synapse sql pool pause --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool pause --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool resume'] = """
 type: command
 short-summary: Resume a SQL pool.
-long-summary: Resume a SQL pool with SQL pool name.
 examples:
   - name: Resume SQL pool.
     text: |-
-        az synapse sql pool resume --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool resume --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool delete'] = """
 type: command
 short-summary: Delete a SQL pool.
-long-summary: Delete a SQL pool with SQL pool name.
 examples:
   - name: Delete SQL pool.
     text: |-
-        az synapse sql pool delete --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool delete --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool wait'] = """
 type: command
 short-summary: Place the CLI in a waiting state until a condition of a SQL pool is met.
 """
 
+helps['synapse workspace firewall-rule'] = """
+type: group
+short-summary:  Manage a workspace's firewall rules.
+"""
+
+helps['synapse workspace firewall-rule create'] = """
+type: command
+short-summary: Create a firewall Rule.
+examples:
+  - name: Create a firewall Rule.
+    text: |-
+        az synapse workspace firewall-rule create --name allowAll --workspace-name testsynapseworkspace \\
+        --resource-group rg --start-ip-address 0.0.0.0 --end-ip-address 255.255.255.255
+"""
+
+helps['synapse workspace firewall-rule show'] = """
+type: command
+short-summary: Get a firewall rule.
+examples:
+  - name: Get a firewall rule.
+    text: |-
+        az synapse workspace firewall-rule show --name rule1 --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace firewall-rule list'] = """
+type: command
+short-summary: List all firewall rules.
+examples:
+  - name: List all firewall rules.
+    text: |-
+        az synapse workspace firewall-rule list --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace firewall-rule delete'] = """
+type: command
+short-summary: Delete a firewall rule.
+examples:
+  - name: Delete a firewall rule.
+    text: |-
+        az synapse workspace firewall-rule delete --name rule1 --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace firewall-rule wait'] = """
+type: command
+short-summary: Place the CLI in a waiting state until a condition of a firewall rule is met.
+"""
+
 helps['synapse spark batch'] = """

```

Comment:
>synapse spark batch [](start = 7, length = 19)

What do you think if we change the command group to `az synapse spark job`? I think `job` is more direct and accurate. You see, the Synapse Studio uses the term "Spark job definition" rather than "Spark batch definition". Besides, `job` is consistent with PowerShell cmdlets.

We can disucuss offline about this. `batch` also looks good to me. We might need to consult with our PM.

Reply:
I think that `az synapse spark job` is ok. Because I know batch from livy Api.
As Both  Synapse studio and Powershell use job. I agree with chaning it. I can change it firstly then consult with PM.

---
In reply to: [418037816](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418037816) [](ancestors = 418037816)